package.json

```diff
@@ -8,16 +8,18 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "lint": "npx eslint src",
-    "lint:fix": "npm run lint --fix",
+    "lint:fix": "npm run lint -- --fix",
```

Comment:
Retornou os -- é isso mesmo?

Reply:
Yes, I set again this to pass "--fix" in npm command, if not throw a error in terminal